terraform/applications/moderator/mysql.tf

```diff
@@ -33,10 +33,10 @@ resource "aws_db_instance" "moderator" {
   max_allocated_storage       = var.mysql_storage_max
   multi_az                    = var.environment == "prod" ? true : false
   allow_major_version_upgrade = true
-  name                        = var.project
```

Comment:
Do we not need the name anymore?

Reply:
yes - the name argument was deprecated 
│   on mysql.tf line 36, in resource "aws_db_instance" "moderator":
│   36:   name                        = var.project
│ 
│ An argument named "name" is not expected here.